### PR TITLE
Curly braces error in simple.md

### PR DESCRIPTION
**Pull request by MdSahil-oss**
This is a documentation error fix, where curly braces were not matching.

### CODE REVIEW

File: docs/source/deploying/simple.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ location ~ / {
   add_header Cache-Control "public";
   expires +1m;
   proxy_pass http://volto;
+  }
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:68]: Please use consistent 4-space indentation throughout this example. [nginx examples use 4 spaces for indentation](https://docs.nginx.com/nginx/admin-guide/web-server/web-server/#virtual-server), and so should we.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Your fix also made me notice indentation inconsistencies in the example. Please fix them throughout this example using 4 spaces per nested level. Thank you!

This PR will close https://github.com/plone/volto/issues/3157

File: docs/source/deploying/simple.md (hunk 1 above)
Comment (MdSahil-oss) [reply docs/source/deploying/simple.md]: Done! @stevepiercy

File: docs/source/deploying/simple.md [hunk 2]
```diff
@@ -62,10 +62,10 @@ location ~ /api($|/.*) {
 location ~ / {
   # Default set to 1m - this is mainly to make PSI happy, adjust to your needs
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:63]:
```suggestion
    # Default set to 1m - this is mainly to make PSI happy, adjust to your needs
```

File: docs/source/deploying/simple.md [hunk 3]
```diff
@@ -62,10 +62,10 @@ location ~ /api($|/.*) {
...
   location ~* \.(ico|jpg|jpeg|png|gif|svg|js|jsx|css|less|swf|eot|ttf|otf|woff|woff2)$ {
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:64]:
```suggestion
    location ~* \.(ico|jpg|jpeg|png|gif|svg|js|jsx|css|less|swf|eot|ttf|otf|woff|woff2)$ {
```

File: docs/source/deploying/simple.md [hunk 4]
```diff
@@ -62,10 +62,10 @@ location ~ /api($|/.*) {
...
-  add_header Cache-Control "public";
-  expires +1m;
-  proxy_pass http://volto;
-  }
+    add_header Cache-Control "public";
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:65]:
```suggestion
        add_header Cache-Control "public";
```

File: docs/source/deploying/simple.md [hunk 5]
```diff
@@ -62,10 +62,10 @@ location ~ /api($|/.*) {
...
+    expires +1m;
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:66]:
```suggestion
        expires +1m;
```

File: docs/source/deploying/simple.md [hunk 6]
```diff
@@ -62,10 +62,10 @@ location ~ /api($|/.*) {
...
+    proxy_pass http://volto;
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:67]:
```suggestion
        proxy_pass http://volto;
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Use proper indentation for nesting {}

**Comment by stevepiercy** [comment]
@MdSahil-oss Use proper indentation for nesting `{}`

**Comment by MdSahil-oss** [comment]
> @MdSahil-oss Use proper indentation for nesting `{}`

@stevepiercy I think, Now it's good to go.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Indentation is still not correct.

File: .gitignore [hunk 7]
```diff
@@ -69,3 +69,6 @@ public/critical.css
 # Sphinx and MyST
 docs/_build/
 /.python-version
+
+# gitpod
+.gitpod.yml
```
Comment (stevepiercy) [inline .gitignore:74]: This is not your user's .gitignore. Please revert this change, and add it to `~/.gitignore_global`.

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,8 @@
 
 ## 16.0.0 (unreleased)
 
+- Updated simple.md @MdSahil-oss
```
Comment (stevepiercy) [inline CHANGELOG.md:5]: This is not under the correct subheading. Please move under this version's Documentation subheading.

File: docs/source/deploying/simple.md [hunk 9]
```diff
@@ -60,11 +60,12 @@ location ~ /api($|/.*) {
 }
 
 location ~ / {
-  # Default set to 1m - this is mainly to make PSI happy, adjust to your needs
-  location ~* \.(ico|jpg|jpeg|png|gif|svg|js|jsx|css|less|swf|eot|ttf|otf|woff|woff2)$ {
-  add_header Cache-Control "public";
-  expires +1m;
-  proxy_pass http://volto;
+# Default set to 1m - this is mainly to make PSI happy, adjust to your needs
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:63]:
```suggestion
    # Default set to 1m - this is mainly to make PSI happy, adjust to your needs
```

File: docs/source/deploying/simple.md [hunk 10]
```diff
@@ -60,11 +60,12 @@ location ~ /api($|/.*) {
...
+    location ~* \.(ico|jpg|jpeg|png|gif|svg|js|jsx|css|less|swf|eot|ttf|otf|woff|woff2)$ {
+        add_header Cache-Control "public";
+        expires +1m;
+        proxy_pass http://volto;
+  }
```
Comment (stevepiercy) [inline docs/source/deploying/simple.md:68]:
```suggestion
    }
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by MdSahil-oss** [comment]
@stevepiercy I think now everything is good to go.

**Review by stevepiercy — APPROVED** [review]
